client/src/pages/private/ParticipantTable.js

```diff
@@ -401,6 +417,13 @@ const ParticipantTable = () => {
         );
       case 'postHireStatuses':
         return getGraduationStatus(row[columnId] || []);
+      case 'engagedBy':
+        return (
+          <>
+            <div className={classes.cellTextStrong}>{row.engagedBy}</div>
```

Comment:
Can we replace _div_ either with `Box` or `Typography`? I feel like for `Typography` we already have some predefined styles, so there's no need to introduce `cellTextStrong`

Reply:
My only concern with Typography is that it won't introduce a line break, but I'll do some experimenting with Typography and Box to confirm and find something that works- I'll get rid of the div.